src/py42/clients/securitydata.py

```diff
@@ -185,6 +185,25 @@ def search_file_events(self, query):
         """
         return self._file_event_service.search(query)
 
+    def get_all_file_events(self, query, page_token=""):
+        """Searches for all file events.
+        `REST Documentation <https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Forensic_File_Search_API>`__
+
+        Args:
+            query (:class:`py42.sdk.queries.fileevents.file_event_query.FileEventQuery`): Also
```

Comment:
This is feedback for both this param description and the one in `search_file_events()`:

The type should be 

```
`:class:`py42.sdk.queries.fileevents.file_event_query.FileEventQuery` or str
```

And the description should just be a description and not mention the type, like

```
The file event query to filter search results.
```

Reply:
Done.